blocks/bricks/__init__.py

```diff
@@ -658,10 +658,17 @@ def add_property(f):
         return add_property
 
     def __getattr__(self, attr):
+        # Prevents infinite recursion when no attribute "f" is available
+        # (happens during serialization under unknown circumstances)
+        try:
+            self_f = self.__getattribute__("f")
```

Comment:
But when `__getattr__` is called for an attribute which is indeed not present, the map `self.f` is requested. If it is not present as well, infinite loop starts.